src/main/java/com/example/demo/core/Pair.java

```diff
@@ -0,0 +1,40 @@
+package com.example.demo.core;
+
+public class Pair<K, V> {
+
+    private K first;
+    private V second;
+
+    public static <K, V> Pair<K, V> createPair(K element0, V element1) {
```

Comment:
element0 i 1? to wygenerowana metoda? wszędzie indziej masz first, second

Reply:
metoda pochodzi z szablonu, zapomniałem ją usunąć